chrome/background.js

```diff
@@ -21,6 +21,17 @@ function fillLoginForm(login) {
       });
     }
   );
+
+  chrome.tabs.executeScript(
+    {
+      file: "/injecttop.js"
```

Comment:
For clarify let's be specific and just call this file `inject_totp.js` or something like that - we can always rename if there will be another use for this file later.